region/brinstar/red.json

```diff
@@ -272,7 +272,13 @@
                 {
                   "name": "Base",
                   "notable": false,
-                  "requires": []
+                  "requires": [
+                    {"or": [
+                      "HiJump",
+                      "h_canFly",
+                      "canWalljump"
```

Comment:
SpringBall jumps work here, or a 68-frame shinespark (dead-easy execution, only needs ~30 frames remaining coming in from node 1).